draft-ietf-quic-version-negotiation.md

```diff
@@ -154,17 +224,17 @@ sends a new first flight with that version - we refer to this version as the
 
 The new first flight will allow the endpoints to establish a connection using
 the negotiated version. The handshake of the negotiated version will exchange
-handshake version information (see {{hs-vers-info}}) required to ensure that VN
-was genuine, i.e. that no attacker injected packets in order to influence the
-VN process, see {{downgrade}}.
+Version Information (see {{vers-info}}) required to ensure that VN was genuine,
```

Comment:
Capitalizing this suggests it's referring to the structure, but it's the information within that's relevant.
```suggestion
information (see {{vers-info}}) used to ensure that VN was genuine,
```

Reply:
Fixed